src/commands.py

```diff
@@ -478,25 +469,10 @@ def _get_user_for_reply(message: discord.Message, message_forwarder: MessageForw
     # If it's a reply thread, the user the reply thread is for, otherwise None
     thread_user = message_forwarder.get_userid_for_user_reply_thread(message)
 
-    # If given '^' instead of user, message the last person to DM bouncer
-    # Uses whoever DMed last since last startup, don't bother keeping in database or anything like that
-    if message.content.split()[1] == "^":
```

Comment:
Maybe bouncer should strip a leading '^' before sending the message, or throw a `GetUserForReplyException` here, so staff doesn't accidentally send messages to users that start with `^`?

Reply:
It's a good idea. I went ahead and raised an exception there, so that it isn't silently changing people's messages.